boards/DCM/Test/Src/Test_StateMachine.cpp

```diff
@@ -109,8 +109,8 @@ TEST_F(
 {
     // We need to tick twice, once to run the `Init` state, and once more
     // to have the state machine transition to the `Run` state.
-    App_SharedStateMachine_Tick(state_machine);
-    App_SharedStateMachine_Tick(state_machine);
+    App_SharedStateMachine_Tick1Hz(state_machine);
```

Comment:
I wonder how our `Test_StateMachine` is going to accommodate for different tick rates. For example, should this test be ticking `1Hz` or `1Khz` or both? It seems a bit tricky to me.

Reply:
Yeah, we're probably going to just end up testing one or the other.